insights/core/evaluators.py

```diff
@@ -288,11 +288,11 @@ def run_reducers(self, metadata=None):
             self.parser_results,
             self.all_output,
             self.handle_reducer_error(),
-            stats=self.stats['reducer'])
+            reducer_stats=self.stats['reducer'])
```

Comment:
I'm not sure, but there's a good chance @csams intentionally renamed this to `stats` since `reducer_stats` doesn't make sense anymore.  I think leaving it as `stats` would be fine, though this might mean changing it on the 1.x branch.